lib/config/dpl-cms/dplCmsConfig.ts

```diff
@@ -9,18 +11,27 @@ const queryDplCmsConfig = async () => {
 }
 
 export const getDplCmsUniloginConfig = async () => {
+  const wellknownUrl = getServerEnv("UNILOGIN_WELLKNOWN_URL")
+  const clientId = getServerEnv("UNILOGIN_CLIENT_ID")
+  const clientSecret = getServerEnv("UNILOGIN_CLIENT_SECRET")
+
+  // If all env vars are present, return them directly without fetching config
+  if (wellknownUrl && clientId && clientSecret) {
+    return {
+      wellknownUrl,
+      clientId,
+      clientSecret,
+      apiData: null,
+    }
+  }
```

Comment:
I still think that we can do this in a partial way / merging values from env and fetch.
How about that we:
* Get the values as you do in line 14-16
*.Make an object out of it and eg. call it `configEnv`
* Fetch the API config with a try/catch block so we do not react on network/malformed errors
* Create an object with the API values eg. call it `configAPI`. If an error occured fetching the data the object will be empty
* Return `{...configAPI, ...configEnv}`

Could that make sense?